core/unix/os_exports.h

```diff
@@ -78,16 +78,22 @@
 #    if defined(MACOS64)
 #        define SEG_TLS SEG_GS     /* DR is sharing the app's segment. */
 #        define LIB_SEG_TLS SEG_GS /* libc+loader tls */
+#        define STR_SEG "gs"
+#        define STR_LIB_SEG "fs"
```

Comment:
Based on LIB_SEG_TLS, shouldn't this be gs?

Reply:
True, it should be gs. Funnily enough previous version of this log used "fs" and I presume it's a mistake.